src/assets/css/style.css

```diff
@@ -1,9 +1,149 @@
 body {
```

Comment:
why are you giving measurements and properties to the whole body. Are you not just working on buttons?